snap/snapcraft.yaml

```diff
@@ -560,7 +560,7 @@ parts:
         reference="$(echo ${line} | cut -f3 -d',')"
         git clone "${repository}" -b "${reference}" "addons/${name}"
       done
-      echo "${ADDONS_REPOS_ENABLED}" > addons/.auto-add
+      echo "core" > addons/.auto-add
```

Comment:
I'd rather this was changed in [set-env-variables.sh] instead (https://github.com/ubuntu/microk8s/blob/9536527e31d0d4d9c59f529d247c152625e68af5/build-scripts/set-env-variables.sh#L56)